<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistContainerView</class>
 <widget class="QWidget" name="PlaylistContainerView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Playlist</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTabWidget" name="widgetsContainer">
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="tabsClosable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionAddFiles">
   <property name="icon">
    <iconset>
     <normalon>:/playlist/addfiles.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Add files</string>
   </property>
  </action>
  <action name="actionDeepScan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Deep scanning</string>
   </property>
  </action>
  <action name="actionCreatePlaylist">
   <property name="icon">
    <iconset>
     <normalon>:/playlist/create.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Create playlist</string>
   </property>
  </action>
  <action name="actionLoadPlaylist">
   <property name="icon">
    <iconset>
     <normalon>:/playlist/open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open playlist</string>
   </property>
  </action>
  <action name="actionSavePlaylist">
   <property name="icon">
    <iconset>
     <normalon>:/playlist/save.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save playlist</string>
   </property>
  </action>
  <action name="actionClosePlaylist">
   <property name="icon">
    <iconset>
     <normalon>:/playlist/close.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Close playlist</string>
   </property>
  </action>
  <action name="actionClearPlaylist">
   <property name="icon">
    <iconset>
     <normalon>:/playlist/clear.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Clear playlist</string>
   </property>
  </action>
  <action name="actionLoop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/playlist/looped.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Loop</string>
   </property>
  </action>
  <action name="actionRandom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/playlist/random.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Random</string>
   </property>
  </action>
  <action name="actionAddFolder">
   <property name="icon">
    <iconset>
     <normalon>:/playlist/addfolder.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Add folder</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../res/icons.qrc"/>
 </resources>
 <connections/>
</ui>
